app/javascript/home/Home.vue

```diff
@@ -77,6 +77,16 @@ function setScrollToFragmentTimeouts() {
 </script>
 
 <style lang="scss">
+:root {
+  --user-header-height: calc(var(--spacing) * 8);
```

Comment:
This is the same value as the `.h-8` on the logged-in header. Hopefully I will never change that without thinking to also update this...